modules/gitops-accessing-gitops-monitoring-dashboards.adoc

```diff
@@ -24,9 +24,8 @@ Disabling or changing the content of the dashboards is not supported.
 
 . In the *Administrator* perspective of the web console, go to *Observe* -> *Dashboards*.
 
-. From the *Dashboard* drop-down list, select the desired {gitops-shortname} dashboard: *{gitops-shortname} (Overview)*, *{gitops-shortname} / Components*, or *{gitops-shortname} /  gRPC Services*.
+. From the *Dashboard* drop-down list, select the desired {gitops-shortname} dashboard: *{gitops-shortname} (Overview)*, *{gitops-shortname} / Components*, or *{gitops-shortname} /  gRPC Services* or *{gitops-shortname} /  Rollouts*.
```

Comment:
```suggestion
. From the *Dashboard* drop-down list, select the desired {gitops-shortname} dashboard: *{gitops-shortname} (Overview)*, *{gitops-shortname} / Components*, *{gitops-shortname} /  gRPC Services* or *{gitops-shortname} /  Rollouts*.
```

Reply:
Yes, this feedback makes sense. I have applied the change in the content, @skrthomas. Please merge the update if everything looks fine. Thank you.